src/redux/studio.js

```diff
@@ -111,10 +120,32 @@ const getRoles = (studioId, username, token) => (dispatch => {
     });
 });
 
+// Selectors
+
+// Fine-grain selector helpers - not exported, use the higher level selectors below
+const isCreator = state =>
```

Comment:
Does it make sense to name these to start with select, or remove the "select" part of "selectIsAdmin, selectIsSocial" for naming consistency?

Reply:
Yeah I didn't name them in the convention just because they are helpers and I wanted to avoid the temptation to export them.